docs/UserGuide.md

```diff
@@ -238,6 +240,7 @@ Format: `delete INDEX1, INDEX2, INDEX3, …`
 * The indices do not need to be listed in order. (i.e. `2, 4, 6` is the same as `6, 2, 4`)
 * Listing the same index more than once will result in an error message being displayed.
 * Each `INDEX` must still adhere to the points listed above.
+* Whitespace between indices is not important. i.e. `delete 1,2,3` is the same as `delete 1, 2, 3`.
```

Comment:
We need to balance carefully between being informative and not throwing a wall of text to the user instead of blindly following bug reporter.

I argue that explicitly mentioning this doesn't seem to be a particularly good idea. (After all, if the user try it out, it will be obvious)